module/mailer/src/Resources/config/services.yml

```diff
@@ -21,5 +21,6 @@ services:
 
     Ergonode\Mailer\Infrastructure\Sender\Strategy\SymfonyMailerStrategy:
         arguments:
+            $localeAware: '@translator.data_collector'
```

Comment:
U can setup default property in autowire, because it was used more than once

Reply:
I wouldn't do that. It's in different modules, would have to be done on some 'common' ground, not SF default and since we'll be probably removing autowiring at all makes no sense